packages/ckeditor5-list/tests/list/utils.js

```diff
@@ -444,10 +449,31 @@ describe( 'utils', () => {
 			);
 
 			expect( getSiblingNodes( document.selection.getFirstPosition(), 'forward' ) ).to.deep.equal( [
+				document.getRoot().getChild( 2 ),
 				document.getRoot().getChild( 3 ),
 				document.getRoot().getChild( 4 )
 			] );
 		} );
+
+		it( 'should return only list items from block quoted list when selection is inside (direction="forward")', () => {
```

Comment:
Is there a similar test but for direction = "backward"? I know we didn't fix that case but it would be good to have symmetry in test, in case if someone changes something in the future.